From: KevinDuan <duankaiwen@kylinos.cn>
Date: Fri, 21 Aug 2026 03:42:14 +0000
Subject: Translated using Weblate (French)

Currently translated at 100.0% (11 of 11 strings)

Translation: openkylin-huanghe/peony-drive-rename
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-drive-rename/fr/
---
 .../translations/peony-drive-rename_fr.ts          | 101 +++++++++++++++++++++
 1 file changed, 101 insertions(+)
 create mode 100644 peony-drive-rename/translations/peony-drive-rename_fr.ts

diff --git a/peony-drive-rename/translations/peony-drive-rename_fr.ts b/peony-drive-rename/translations/peony-drive-rename_fr.ts
new file mode 100644
index 0000000..8793135
--- /dev/null
+++ b/peony-drive-rename/translations/peony-drive-rename_fr.ts
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="fr">
+<context>
+    <name>Peony::DriveRename</name>
+    <message>
+        <source>drive rename</source>
+        <translation type="vanished">设备重命名</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="207"/>
+        <location filename="../drive-rename.cpp" line="223"/>
+        <source>Rename</source>
+        <translation>Renommer</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="224"/>
+        <source>Device name:</source>
+        <translation>Nom de l&apos;appareil :</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="227"/>
+        <source>OK</source>
+        <translation>Confirmer</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="228"/>
+        <source>Cancel</source>
+        <translation>Annuler</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="236"/>
+        <location filename="../drive-rename.cpp" line="243"/>
+        <location filename="../drive-rename.cpp" line="250"/>
+        <source>Warning</source>
+        <translation>Avertissement</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="236"/>
+        <source>Renaming cannot start with a decimal point, Please re-enter!</source>
+        <translation>Le renommage ne prend pas en charge le point comme premier caractère. Veuillez saisir à nouveau !</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="243"/>
+        <source>The device name exceeds the character limit, rename failed!</source>
+        <translation>Le nom de l&apos;appareil dépasse la limite de caractères. Échec du renommage !</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="250"/>
+        <source>Renaming will unmount the device. Do you want to continue?</source>
+        <translation>Le renommage désinstallera l&apos;appareil. Voulez-vous continuer ?</translation>
+    </message>
+    <message>
+        <source>The device may not support the rename operation, rename failed!</source>
+        <translation type="vanished">可能设备不支持重命名操作，重命名失败！</translation>
+    </message>
+    <message>
+        <source>Failed to rename!</source>
+        <translation type="vanished">重命名失败！</translation>
+    </message>
+</context>
+<context>
+    <name>Peony::DriveRenamePlugin</name>
+    <message>
+        <source>drive rename</source>
+        <translation type="vanished">设备重命名</translation>
+    </message>
+    <message>
+        <location filename="../driverenameplugin.h" line="41"/>
+        <source>Drive rename</source>
+        <translation>Renommer l&apos;appareil</translation>
+    </message>
+</context>
+<context>
+    <name>Peony::DriverAction</name>
+    <message>
+        <source>Send to a removable device</source>
+        <translation type="vanished">发送到移动设备</translation>
+    </message>
+</context>
+<context>
+    <name>Peony::SendToPlugin</name>
+    <message>
+        <source>Send to a removable device</source>
+        <translation type="vanished">发送到移动设备</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../drive-rename.cpp" line="418"/>
+        <source>Warning</source>
+        <translation>Avertissement</translation>
+    </message>
+    <message>
+        <location filename="../drive-rename.cpp" line="418"/>
+        <source>The device may not support the rename operation, rename failed!</source>
+        <translation>Il est possible que l&apos;appareil ne prenne pas en charge l&apos;opération de renommage. Échec du renommage !</translation>
+    </message>
+</context>
+</TS>
